specs/feature-mouse-interactions/design.md

```
@@ -110,6 +110,40 @@ The core flow for all interaction commands is: resolve target → get element co
 6. Output result
 ```
 
+### Data Flow — `interact click s1 --hold Shift`
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)